packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -29,24 +36,59 @@ export class SObjectDescribe {
    * @param type SObjectCategory
    * @returns string[] containing the sobject names
    */
-  public async describeGlobal(type: SObjectCategory): Promise<string[]> {
+  public async describeGlobal(
```

Comment:
Moved the filter logic to this call. Only if we execute the complete flow (query for sobject list, query for all sobject describes, create faux classes, etc) should we filter the sobjects. In the following weeks we'll start introducing flows where we already have the list of sobjects and in those scenarios we don't want to filter things out.

This keeps backwards compatibility with existing usage.